libp2p/builders.nim

```diff
@@ -197,6 +205,11 @@ proc build*(b: SwitchBuilder): Switch
     ms = ms,
     nameResolver = b.nameResolver)
 
+  if b.isCircuitRelay:
+    let relay = Relay.new(switch, b.circuitRelayCanHop)
+    switch.mount(relay)
+    switch.addTransport(RelayTransport.new(relay, muxedUpgrade))
```

Comment:
Actually, now that this is cleaner, we could even remove it from the builder, and let the user build it himself (like gossipsub)